md-commons/src/main/java/com/wipro/ats/bdre/md/beans/table/AppDeploymentQueueStatus.java

```diff
@@ -27,27 +27,27 @@ public AppDeploymentQueueStatus() {
     }
 
     public AppDeploymentQueueStatus(Short adqState, String description) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove the unused method parameter(s) "adqState". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS1172)